frappe/templates/print_formats/standard_macros.html

```diff
@@ -151,7 +151,7 @@
 	{% elif df.fieldtype=="Signature" %}
 		<img src="{{ doc[df.fieldname] }}" class="signature-img img-responsive"
 			{%- if df.print_width %} style="width: {{ get_width(df) }};"{% endif %}>
-	{% elif df.fieldtype in ("Attach", "Attach Image") and frappe.utils.is_image(doc[df.fieldname]) %}
+	{% elif df.fieldtype in ("Attach", "Attach Image") %}
```

Comment:
@s-aga-r 
1. Please add before and after gifs/ images. They help in review and testing.
2. This change would not let any image be displayed if it is on a link.
3. If this issue exists it should either be caught in the is_image function or the link should be separately validated.
4. We usually create PR's in the develop branch and then backport them to version-13-hotfix AND/ or version-12-hotfix as per need

Reply:
3 - when the extension isn't present in link,  is_image can't check if image is valid without downloading it. I don't think it's worth downloading an image just to decide if it's worth rendering as image. 

If you look at history, image fields were rendered as images unconditionally before. This validation is unnecessary IMO.

Reply:
2 - not true. This is ignoring validation and rendering all image link as images.